instructor/models.py

```diff
@@ -168,17 +168,17 @@ class MicroscopyImages(models.Model):
 
 FACS_FIXED = 'Fixed'
 
-FACS_KINDS = (( 'Dye', 'Dye/Stain' ), ('Anti', 'Antibody-labeling'))
+FACS_KINDS = (('', 'Select Analysis'), ('Anti', 'Antibody-labeling'), ('Dye', 'Dye/Stain'))
 
 FACS_DYE = 'Dye'
```

Comment:
Nit: If `FACS_FIXED` and `FACS_DYE` are meant to be keys, it might be clearer to use them in FACS_KINDS